code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -4197,3 +4195,89 @@
 /obj/item/weapon/reagent_containers/food/snacks/frog_leg/New()
 	..()
 	reagents.add_reagent(NUTRIMENT, 6)
+
+
+/obj/item/weapon/reagent_containers/food/snacks/pie/welcomepie
+	name = "friendship pie"
+	desc = "Offered as a gesture of Vox goodwill." //"Goodwill"
+	icon_state = "welcomepie"
+	bitesize = 4
+	New()
+		..()
+		reagents.add_reagent("sacid",6)
+		reagents.add_reagent("nutriment",2)
+
+/obj/item/weapon/reagent_containers/food/snacks/zhulongcaofan
+	name = "zhu l�ng cao f�n"
+	desc = "Literally meaning 'pitcher plant rice'. After carefully cleansing and steaming the pitcher plant, it is stuffed with steamed rice. The carnivorous plant is rich with minerals from fauna it has consumed."
+	icon_state = "zhulongcaofan"
+	bitesize = 3
+	New()
+		..()
+		reagents.add_reagent("nutriment",6)
```

Comment:
Ugh, I had them changed. I think the rebase lost the defines somehow. What a pain.